lib/docs/scrapers/sanctuary.rb

```diff
@@ -0,0 +1,25 @@
+module Docs
+
+  class Sanctuary < UrlScraper
+    self.name = "Sanctuary"
+    self.slug = "sanctuary"
+    self.type = "sanctuary"
+    self.release = "3.1.0"
+    self.base_url = "https://sanctuary.js.org/"
+    self.links = {
+      home: "https://sanctuary.js.org/",
+      code: "https://github.com/sanctuary-js/sanctuary",
+    }
```

Comment:
I copied this from the Ramda file. I don't see `links` referenced in the documentation. Let me know whether I should remove this. :)